Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -724,6 +724,74 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Update Once-Conflicted Doc - SGColl", "[.SyncServerCollection]") {
+
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl;
+    const string docID = timePrefix() + "uocd-doc";
+
+    // Create a conflicted doc on SG, and resolve the conflict
+    sendRemoteRequest("PUT", docID + "?new_edits=false",
```

Comment:
It would be better to use the version that has the collection path in the parameter list after 3777 is merged in.